deploy/olm-catalog/performance-addon-operator/4.5.0/performance-addon-operator.v4.5.0.clusterserviceversion.yaml

```diff
@@ -109,6 +109,8 @@ spec:
           strategy: {}
           template:
             metadata:
+              annotations:
+                workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```

Comment:
I don't think we need to add annotations to old (pre 4.7) manifests, but won't hurt either.

Reply:
We should not forget to update d/s manifests.